apps/app/pages/settings.tsx

```diff
@@ -239,6 +239,7 @@ function Settings() {
               onClick={() => {
                 changeSkills();
                 formPersonal.submit();
+                notifyInfo("Atualizado com sucesso!");
```

Comment:
This should be a promise in order to catch a possible error and display it.

Reply:
I thought about at first but then somehow I got to understand that there's no response when using that button 😅 (or at least I'm not understanding it). So, in order to improve the user experience I added the notification. I'll try to use a ```
.catch 
``` for it 🙏

Reply:
You can directly call `edit_user` if that's easy 👌🏼

Reply:
I uploaded some code just now. I have been searching for some hours and I can´t find a way to use a ```.then``` or a ```.catch``` in a function like edit_user that has '(args: any) => void'... :sweat_smile:

Reply:
Yap, it's actually not possible. Those are only usable on functions whose return value is a promise.

Reply:
So once that we have "catch" for errors in the fields of the form, couldn't we just notify as "Atualizado com sucesso!"? Because, if not, there will be a error message from the function failing, am I right? This feature would only prevent from misunderstanding if the user clicked "GUARDAR" or not.